Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -0,0 +1,17 @@
+function targetBlank() {
```

Comment:
To avoid polluting the global namespace, wrap the whole script into an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).